### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by nagayev**
- Add getColor function
- Docs improvements
- Removed data/b.png

### CODE REVIEW

File: pnglib.cpp [hunk 1]
```diff
@@ -175,6 +175,15 @@ void write_png_file( const std::string &filename_, const pngimage_t &image_ ) {
     std::copy(data_to_write.begin(), data_to_write.end(), std::ostreambuf_iterator<char>(f));
 }
 
+unsigned int getColor(int8_t r, int8_t g, int8_t b, int8_t a){
+    unsigned int color = 0b0;
```
Comment (pantadeusz) [inline pnglib.cpp]: Why not just unsigned int color = 0;? or even unsigned color = 0; ?

File: example.cpp [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+using namespace std;
+using namespace puzniakowski::png;
+
+int main(){
+    pngimage_t im;
+    im = read_png_file("data/a.png");
+    for (int i=0;i<im.width;i++){
+        for (int j=0;j<im.height;j++){
+            //set color to R = 0, G = 255, B = 255, A = 0
+            unsigned int color = getColor(0,255,255,0);
```
Comment (pantadeusz) [inline example.cpp]: Is transparent pixel on purpose?

Comment (nagayev) [reply example.cpp]: @pantadeusz I have updated this example, please check it.

File: pnglib.cpp (hunk 1 above)
Comment (nagayev) [reply pnglib.cpp]: Fixed

**Comment by pantadeusz** [comment]
Dear @nagayev . 
After some consideration I am not sure why do we need this helper functions. The values are in one place in memory. Probably using union would solve the problem with low expressiveness of single 32bit number. What do You think about it?

**Comment by nagayev** [comment]
@pantadeusz As I know many cpp beginners don't know about unios. I think these functions can help them.

**Comment by pantadeusz** [comment]
Yes, but unions are extremely useful in situations when we want to access one memory region using multiple approaches (like pixel can be one number like 0x0ffaabbcc or it can be split into channels). Similar approach can be seen in, for example, [SDL](https://wiki.libsdl.org/SDL2/SDL_Event) and it is much more complex data structure than pixel.

**Comment by nagayev** [comment]
@pantadeusz So what is your vision about my edits?
Should I remove `getColor` functions?

**Comment by pantadeusz** [comment]
Ok. I will merge it. Sorry for the delay. I was quite busy and forgot about this thread.

**Comment by pantadeusz** [comment]
BTW - I added You to the contributors in the README. Please look @nagayev

**Comment by nagayev** [comment]
@pantadeusz Thank you.

**Pull request merged**